clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientV2Provider.groovy

```diff
@@ -43,4 +43,9 @@ class OpenstackClientV2Provider extends OpenstackClientProvider {
 
   //TODO v2 specific operations
 
+  @Override
+  String getRegion() {
+    // v2 does not have a way to access the region
+    return ""
```

Comment:
A lot of operations/requests are keyed by region when Gate is doing a lookup. This should probably be something like `"NA"` or `"default"`